index.d.ts

```diff
@@ -1,11 +1,15 @@
 export interface IContactInfo {
     /**
-     * In seconds
+     * How long the contact was in proximity. Minimum duration is 5 minutes and increments by 5 minutes: 5, 10, 15, etc.
+     * 
+     * This value is in seconds.
      */
     duration: number;
 
     /**
-     * In seconds
+     * This property contains the time when the contact occurred. This may have reduced precision, such as within one day of the actual time.
+     * 
+     * This value is in seconds since 00:00:00 UTC on 1 January 2001.
      */
     timestamp: number;
```

Comment:
It might make sense to convert this to a `Date` in the future, but for now at least documenting this reference date should help those that are unfamiliar with Apple’s epoch.